std/format/package.d

```diff
@@ -1341,12 +1341,10 @@ if (isSomeChar!Char)
 }
 
 /// ditto
-typeof(fmt) format(alias fmt, Args...)(Args args)
-if (isSomeString!(typeof(fmt)))
+auto format(alias fmt, Args...)(Args args)
+if (is(StringTypeOf!(typeof(fmt))))
 {
     import std.array : appender;
-    import std.range.primitives : ElementEncodingType;
-    import std.traits : Unqual;
```

Comment:
Why are these imports made public?

Reply:
Sorry, I needed that for stating the return type explicitly and forgot to undo this, when using `auto`.